build/android/generate_xwalk_core_library_aar.py

```diff
@@ -26,21 +26,18 @@ def main():
      'res'),
   )
   files = (
+    (os.path.join(options.target, 'xwalk_core_library',
+        'AndroidManifest.xml'), 'AndroidManifest.xml'),
+    (os.path.join(options.target, 'xwalk_core_library', 'libs',
+        'xwalk_core_library_java.jar'), 'classes.jar'),
     (os.path.join(options.target, 'gen', 'xwalk_core_internal_java', 'java_R',
-                  'R.txt'),
-     'R.txt'),
-    (os.path.join(options.target, 'xwalk_core_library', 'AndroidManifest.xml'),
-     'AndroidManifest.xml'),
-    (os.path.join(options.target, 'lib.java', 'xwalk_core_library_java.jar'),
-     'classes.jar'),
+        'R.txt'), 'R.txt'),
   )
   # This is a list of files that will not be packaged: mostly a blacklist of
   # files within |dirs|.
   exclude_files = (
     os.path.join(options.target, 'xwalk_core_library', 'libs',
-                 'xwalk_core_library_java_app_part.jar'),
-    os.path.join(options.target, 'xwalk_core_library', 'libs',
-                 'xwalk_core_library_java_library_part.jar'),
+        'xwalk_core_library_java.jar'),
```

Comment:
The aar only contain xwalk_core_library_java.jar, developer can't use the aar for embedded mode, so we need generate another aar for shared mode, and push it to company maven repository with classify.